clients/admin-ui/src/features/privacy-notices/PrivacyNoticesTable.tsx

```diff
@@ -83,28 +82,7 @@ export const PrivacyNoticesTable = () => {
       </Flex>
     );
   }
-  if (privacyNotices.length === 0) {
```

Comment:
now that we are always showing notices, and notices are seeded into the db, it didn't seem like we needed this empty state anymore